opentracing_instrumentation/client_hooks/_dbapi2.py

```diff
@@ -161,6 +161,10 @@ def __init__(self, connection, module_name, connect_params):
             connect_params=connect_params
         )
 
+        # Tip suggested here:
+        # https://gist.github.com/mjallday/3d4c92e7e6805af1e024.
+        self._sqla_unwrap = connection
```

Comment:
without a test, impossible to see the impact. Can we reproduce the problem reported in #44 in a test?

Reply:
Not without sqlachemy connecting to a postgres database using psycopg2.